packages/core/src/config/rules.ts

```diff
@@ -24,14 +24,25 @@ export function initRules<T extends Function, P extends RuleSet<T>>(
           return undefined;
         }
 
-        const visitor = rule(ruleSettings);
+        const visitors = rule(ruleSettings);
+
+        if (Array.isArray(visitors)) {
+          return visitors.map((visitor: any) => ({
+            severity: ruleSettings.severity,
+            ruleId,
+            visitor: visitor,
+          }))
+        }
 
         return {
           severity: ruleSettings.severity,
           ruleId,
-          visitor,
+          visitor: visitors,
```

Comment:
Should we rename this prop to `visitors` too?

Reply:
Actually no, because we always return only one visitor or array of visitors. 
Agree that it's little bit confusing here in line 40, because `visitors` variable is an object (only one single visitor). 
Left a comment in code which clarifies that.